apps/src/templates/progress/progressHelpers.js

```diff
@@ -88,10 +93,6 @@ export function getIconForLevel(level, inProgressView = false) {
     return match[1];
   }
 
-  if (level.isUnplugged) {
-    return 'scissors';
-  }
-
   if (level.bonus) {
```

Comment:
Does the bonus icon also need to be moved up?

Reply:
Not in the current implementation. When bonus is true, icon is null in the level data so the checkered flag icon shows correctly. 
<img width="417" alt="Screen Shot 2020-04-20 at 11 05 54 AM" src="https://user-images.githubusercontent.com/12300669/79784383-0162b300-82f7-11ea-961a-02773dfa941e.png">
<img width="488" alt="Screen Shot 2020-04-20 at 11 06 51 AM" src="https://user-images.githubusercontent.com/12300669/79784418-0e7fa200-82f7-11ea-8264-f66a14da473c.png">